cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -42,7 +42,7 @@ import (
 )
 
 const (
-	upgradeManifestTimeout = 1 * time.Minute
+	upgradeManifestTimeout = 5 * time.Minute
```

Comment:
i think it's best that we notify the user about the upgrade timeout in `apply` - in the lines of:
`this operation will timeout in X minutes`
or someone might decide to `ctrl+c` earlier thinking this has hanged.

Reply:
I created https://github.com/kubernetes/kubeadm/issues/914 to track this.